packages/material-ui-docs/package.json

```diff
@@ -37,7 +37,7 @@
     "react-dom": "^16.8.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.2.0",
+    "@babel/runtime": "^7.4.4",
```

Comment:
If we transpile with `@babel/plugin-transform-runtime` then we need a production dependency that is equal or higher to the transform version. Otherwise missing dependencies can occur downstream. Consider the following scenario:

Downstream:
`@material-ui/core`: `4.0.0`
`@babel/runtime`: `7.2.0`
Upstream:
`@babel/plugin-transform-runtime`: `7.2.0`

Now we bump `@babel/plugin-transform-runtime` by a minor. In that minor a `@babel/plugin-transform-runtime` leverages a new helper method that is introduced in `7.3.0`. We release a new version of `@material-ui/core` which is built with `@babel/plugin-transform-runtime@7.3.0` without bumping th `@babel/runtime` dependency entry. Downstream updates `@material-ui/core` but not `@babel/runtime` since the semver range still matches. The downloaded package of `@material-ui/core` has no access to the new helper function and crashes.

I think we got reports of this a couple of times. It's fixable downstream by bumping `@babel/runtime` but wouldn't need intervention if we would also bump the entry of `@babel/runtime`.